app/models/user.rb

```diff
@@ -95,6 +95,9 @@ def prompt_for_2sv?
 
   def event_logs
     EventLog.where(uid: uid).order(created_at: :desc)
+      .joins("LEFT JOIN ip_addresses ON ip_addresses.id = event_logs.ip_address_id")
+      .joins("LEFT JOIN user_agents ON user_agents.id = event_logs.user_agent_id")
+      .select('event_logs.*, ip_addresses.ip_address as ip_address, user_agents.user_agent as user_agent')
```

Comment:
It'd be better to use [`Forwardable`](http://ruby-doc.org/stdlib-2.0.0/libdoc/forwardable/rdoc/Forwardable.html) to delegate the `user_agent` attribute to the joined object.